packages/web/components/complex/transfer.tsx

```diff
@@ -281,6 +282,28 @@ export const Transfer: FunctionComponent<TransferProps> = ({
             <span className="body2 md:caption">{warningMessage}</span>
           </GradientView>
         )}
+        {transferPathHealth && transferPathHealth == "Congested" && (
+          <GradientView
+            className="text-center"
+            gradientClassName="bg-gradient-negative"
+            bgClassName="bg-osmoverse-900"
+          >
+            <span className="body2 text-rust-500 md:caption">
+              {"Network congested. Transfers may take longer than usual."}
+            </span>
+          </GradientView>
+        )}
+        {transferPathHealth && transferPathHealth == "Blocked" && (
+          <GradientView
+            className="text-center"
+            gradientClassName="bg-gradient-negative"
+            bgClassName="bg-osmoverse-900"
+          >
+            <span className="body2 text-rust-500 md:caption">
+              {"Transfer network currently blocked. Check back later."}
```

Comment:
It's a chore we'll have to deal with now, but we need this to be translated in all languages.

Reply:
ah ok, I see how localizations are done. I'll add translations to the language` json` files. OK to just use google translate and input what these sentences translate to?

Reply:
Yeah that works for now. We'll need to periodically ask language speakers to look over new ones as we accumulate them.